xl/event.py

```diff
@@ -220,7 +248,10 @@ class EventManager(object):
         Manages all Events
     """
     def __init__(self, use_logger=False, logger_filter=None, verbose=False):
+        # sacrifice space for simplicity
```

Comment:
This really sacrifices space for a bit of `emit()` speed, as you can always call `_emit` twice (once with `callbacks`, once with `ui_callbacks`).